notebooks/QA_run-2-previous-scenarios-on-dev/qa_model_runs.ipynb

```diff
@@ -98,19 +99,21 @@
    "outputs": [],
    "source": [
     "# Get params only from results JSONs, edit scenario name, save to queue folder\n",
-    "# ⚠️ For v3.3 there is a breaking change to params - we need to change the format of NDG\n",
+    "# ⚠️ WARNING!!! If there have been any breaking changes to params you will have to implement them here\n",
     "\n",
     "if not os.path.exists(\"queue\"):\n",
     "    os.makedirs(\"queue\")\n",
     "\n",
     "filenames = []\n",
     "for trust in results_dict.keys():\n",
     "    params = results_dict[trust][\"results_old\"][\"params\"].copy()\n",
-    "    params[\"scenario\"] = params[\"scenario\"] + \"-test\"\n",
+    "    params[\"scenario\"] = params[\"scenario\"] + \"-testdev\"\n",
     "    params_filename = f\"{params['dataset'] + '-' + params['scenario']}.json\"\n",
     "    params[\"app_version\"] = \"dev\"\n",
     "    params[\"user\"] = \"ds-team\"\n",
     "    params[\"viewable\"] = False\n",
+    "    # ⚠️  FOR v4.2 ONLY: REMOVE LINE BELOW IN NEXT MODEL VERSION TESTING\n",
+    "    params[\"inequalities\"] = {\"level_up\": [], \"zero_sum\": [],\"level_down\": []}\n",
```

Comment:
Just to note we changed this to `params["inequalities"] = {}` for v4.2 QA, given that we decided to keep the empty dict structure instead of adding empty lists below it.